wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -622,10 +626,12 @@ public class ZMUserSession: NSObject {
     func createMLSClientIfNeeded() {
         // TODO: [jacob] refactor out WPB-6198
         if applicationStatusDirectory.clientRegistrationStatus.needsToRegisterMLSCLient {
+            guard let mlsClientID = MLSClientID(user: ZMUser.selfUser(in: syncContext)) else {
+                fatalError("Needs to register MLS client but can't retrieve qualified client ID")
```

Comment:
same as fatalError but it should leave a log file and be sent from user.
```suggestion
                fatal("Needs to register MLS client but can't retrieve qualified client ID")
```